setup.py

```diff
@@ -455,7 +420,6 @@ def find_py(self):
         },
     },
     classifiers=[
-        "Programming Language :: Python :: 2",
```

Comment:
While technically this is correct, I'd leave it for a different patch series; sadly there's lots more of Python 2 stuff around...

Reply:
I think that this section `classifiers` doesn't matter, because we don't ship `subscription-manager` on https://pypi.org

Reply:
Yes, I got carried away a bit. Reverting.